.github/shared/src/readme.js

```diff
@@ -12,6 +12,11 @@ import { Tag } from "./tag.js";
  * @typedef {import('./spec-model.js').ToJSONOptions} ToJSONOptions
  */
 
+/**
+ * Regex to match tag names in readme.md yaml code blocks
+ */
+export const TagMatchRegex = /yaml.*\$\(tag\) ?== ?["']([^']*)["']/;
```

Comment:
```suggestion
export const TagMatchRegex = /yaml.*\$\(tag\) ?== ?["']([^'"]*?)["']/;
```

Just for consistency. This is a bit of a case where you can't be certain that your regex will handle the same starting and ending characters. That being said I feel like `?["'](.*?)["']`  is simpler and should also work.

I love regex hahaha feel free to ignore me. Your test case obviously proves this out.